earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/common/utils.py

```diff
@@ -328,9 +328,12 @@ def RunCmd(os_cmd):
   try:
     if isinstance(os_cmd, str):
       os_cmd = shlex.split(os_cmd)
-    results = subprocess.check_output(os_cmd)
+    p = subprocess.Popen(os_cmd, shell=False,
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.PIPE)
+    results = p.communicate()[1]
```

Comment:
This doesn't seem to check the command's exit status like `check_open` does.  I think you should raise an exception, or print a failure if the exit status is non-zero.

Reply:
I, now, added support for it.

Reply:
Yes, I think this addresses the original question too...